core/aof.go

```diff
@@ -6,28 +6,31 @@ import (
 	"os"
 	"strings"
 
+	dbEngine "github.com/dicedb/dice/IStorageEngines"
 	"github.com/dicedb/dice/config"
+	"github.com/dicedb/dice/object"
```

Comment:
instead of creating another package for object see if we can fit it in `storage_engine` or `core`